README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master.svg?style=svg)](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master)
+[![UniLogin](https://circleci.com/gh/UniLogin/UniLogin/tree/master.svg?style=svg)](https://circleci.com/gh/UniLogin/UniLogin/tree/master)
 
 ![Universal-Login](./docs/source/static/logo.png)
```

Comment:
What about here? Can we change this too?

Reply:
Yes, we can 😉